### ISSUE TITLE
Current state of the library

### ISSUE THREAD

**Issue by Goldziher** [willryan/factory.ts#47]
Hi @willryan ,

Thanks for this great library. Is it actively maintained?

I would like to help maintain it - im using it for several projects and there are functionalities I would like to add etc.

**Comment by willryan** [comment]
Hi @Goldziher,

Basically, it's maintained whenever I get a chance (which isn't often).  I have been meaning to get back to it and at least update the Typescript version. Plus probably the top 1 or 2 issues in terms of value for effort.

I'm totally open to Pull Requests. 😄

**Comment by Goldziher** [comment]
> Hi @Goldziher,
> 
> Basically, it's maintained whenever I get a chance (which isn't often). I have been meaning to get back to it and at least update the Typescript version. Plus probably the top 1 or 2 issues in terms of value for effort.
> 
> I'm totally open to Pull Requests. 😄

@willryan - I added an initial dependencies update PR. If you approve and merge, I will add another PR with eslint updates - and if you grant me some rights, I will also setup a github pipeline instead of circle CI. Once that is in place, I will work to increase test coverage -- and then add some new features and functionalities in followup PRs.
